### PR TITLE
Binding desc bd leathf29

### PR DESCRIPTION
**Pull request by ElianaDalSasso**
Here is another BindingDesc. Thank you @thea-m for the Git Hub support!
The file has been:
 - updated in the Binding Desc
 - updated in condition
the previous descriptions were done by Susanne.

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethf29.xml [hunk 1]
```diff
@@ -715,34 +718,54 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                                <decoNote xml:id="b2" type="bindingMaterial">The textile inlays are
-                                    either cotton or silk (see <ref target="#e3"/>). <material
-                                        key="wood"/>
-                                    <material key="leather"/>
-                                    <material key="textile"/>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1" type="bindingMaterial">
+                                  The book is bound in <material key="wood">wooden</material>
+                                  boards covered with blind-tooled <material key="leather"/>.
+                                  The lower board is decorated with a <material key="textile"/> inlay.
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethf29.xml:725]: The keyword textileInlay exists now and could be used here, I think

File: OxfordBodleian/Juel-Jensen/BDLaethf29.xml [hunk 2]
```diff
@@ -715,34 +718,54 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                  A niche possibly for a <term key="mirror"/> is carved into the inner side of
+                                  the upper board (the mirror is lost, but traces of adhesive are still visible).
+                                  The inner side of the lower board is decorated with a silk (or cotton) inlay coloured gold, blue, and
+                                  red (see <ref target="#e3"/>).
+                                </decoNote>
+                                <decoNote xml:id="b3" type="Cover" color="reddish brown">
+                                  Full reddish-brown leather cover.
+                                  The turn-ins have been trimmed out and, at the corners, they form butt mitres.
+                                  <term key="additionalLeatherPatch">Additional leather patches</term> reduce the size
+                                  of the area among the turn-ins.
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethf29.xml:739]: the area between the turn-ins?

File: OxfordBodleian/Juel-Jensen/BDLaethf29.xml [hunk 3]
```diff
@@ -715,34 +718,54 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                  Blind-tooled ornaments decorate the cover in full, including turn-ins, edges, and spine.
+                                  The cover sides present a slightly different design consisting of a rectangular frame of <term key="crissCross"/>
+                                  motifs between <term key="tripleStraightLine">triple straight lines</term> and
+                                  <term key="doubleStraightLine">double straight lines</term>, enclosing a central panel
+                                  with a star. In the areas above and below the star are triangular-shaped patterns formed by superimposed
+                                  <term key="VForm"/> motifs with <term key="doubleCircle">double circles</term> at the top.
+                                  The star is formed by V-form motis and double circles. The lower board present a 4-pointed star while
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethf29.xml:746]: presents?

**Review by thea-m — CHANGES_REQUESTED** [review]
Great, thank you!

File: OxfordBodleian/Juel-Jensen/BDLaethf29.xml (hunk 1 above)
Comment (ElianaDalSasso) [reply OxfordBodleian/Juel-Jensen/BDLaethf29.xml:725]: From now on I will use it!

**Comment by gnizla** [comment]
Thanks @ElianaDalSasso

**Pull request merged**